EcommerceUI/src/app/checkout/checkout.component.ts

```diff
@@ -120,11 +120,9 @@ export class CheckoutComponent implements OnInit {
               } else {
                 this.orderInfo.billingAddress = this.billingAddress;
               }
-              console.log(this.orderInfo);
               //call REST API via the CheckoutService
-              this.cartService.checkOut(this.orderInfo).subscribe(data => {
-                console.log(data);
-              })
+              console.log("Attempting checkout...");
```

Comment:
You seem to have accidentally removed the checkout api call - please readd it.
Optionally you could check the subscribed data for errors & print to the page.